bindings/java/mongocrypt/src/main/java/com/mongodb/crypt/capi/MongoCryptImpl.java

```diff
@@ -232,19 +233,27 @@ public MongoCryptContext createDataKeyContext(final String kmsProvider, final Mo
         }
 
         if (kmsProvider.equals("aws")) {
```

Comment:
Optional: You can remove the "aws" and "local" cases. `mongocrypt_ctx_setopt_key_encryption_key` supports all masterkey formats.